tools/docker/Dockerfile.monorepo

```diff
@@ -8,28 +8,31 @@ ARG PNPM_VERSION
 
 ENV LANG=en_US.UTF-8 \
     LC_ALL=en_US.UTF-8 \
-    JETPACK_MONOREPO_ENV=1
+    JETPACK_MONOREPO_ENV=1 \
+    DEBIAN_FRONTEND=noninteractive \
+    PNPM_HOME=/usr/local/pnpm \
+    PATH="/usr/local/pnpm:${PATH}" \
+    npm_config_update_notifier=false
```

Comment:
I'll update the PR description. This snuck in. It's to disable the prompt to update `pnpm` when running `jp` commands. Since each container is destroyed after use, even if someone updated `pnpm` within the docker container, the image would still have 9.15.0.

I struggled to find documentation of this related to pnpm, but npm added it via https://github.com/npm/cli/pull/1632 and pnpm claims to follow npm's update notification framework. In testing, it _seemed_ to work. At worst, it does nothing.